content/blogposts/2024/sourcegraph-release-june-2024.md

```diff
@@ -0,0 +1,85 @@
+
+This update also enables two [recently announced](https://sourcegraph.com/blog/cody-enterprise-june-2024) Cody Enterprise features: GPT-4o support and increased context windows for better codebase awareness.
+<br />
+
+### Render Jupyter Notebooks and Mermaid diagrams in the file view
+
+<br />
+The Code Search file view now supports rendering Jupyter Notebooks (`.ipynb` files) and Mermaid diagrams (within `.md` files). 
+
+Jupyter Notebook files were previously rendered as JSON, but they’ll now render as rich text and media right in the file view, similar to how they’re rendered in the native Jupyter application.
+
+<br />
+<Video
+  source={{
+    mp4: 'blog/code-search-now-supports-jupyter-notebook-rendering/video_001'
+  }}
+  controls={true}
+  loop={true}
+  title="Searching by the new point-in-time search filter"
+/>
+<br />
+
+[Mermaid](https://mermaid.js.org/) is a JavaScript-based diagramming and charting tool. You can use it to create visualized diagrams, flowcharts, and user journeys directly within Markdown files. Code Search now recognizes [Mermaid syntax](https://mermaid.js.org/intro/syntax-reference.html#syntax-structure) within your Markdown files and will render your diagrams within the file view.
+
+<br />
+<Figure
+  src="https://storage.googleapis.com/sourcegraph-assets/blog/release-post/june-2024/mermaid-file-view.png"
+  alt="Code Search file view showing the Mermaid chart"
+/>
+<br />
+<br />
+
+### GPT-4o support for Cody chat and commands
+
+<br />
+You can now use GPT-4o as your LLM for powering Cody chat and commands. GPT-4o is OpenAI’s latest model, and it’s two times faster than GPT-4 Turbo while performing better in general reasoning benchmarks. 
+
+ [Read the docs](https://sourcegraph.com/docs/cody/core-concepts/cody-gateway#configuring-custom-models) for information on configuring your model selection. You can also try out GPT-4o and compare it against other models today using the [LLM Litmus Test](https://s0.dev/).
+ <br />
+
+### Smart context windows for better codebase awareness
+
+<br />
+Cody has increased context window limits for both inputs and outputs. These changes provide several benefits:
+
+* You can now push way more context into Cody, including @-mentioning multiple large files, so that you can ask questions about larger amounts of code
+* You can have much longer back-and-forth chats with Cody before it starts to forget the context from earlier in the conversation
+* Cody’s outputs should not cut off mid-message anymore
+
+<br />
+For BYOK customers, increasing context limits may increase LLM costs. [Read the docs](https://sourcegraph.com/docs/cody/clients/enable-cody-enterprise#smart-context-window) for more information on this feature and how to set your own context limits.
+
+<br />
+Sourcegraph 5.X.X is now available. Users self-hosting Sourcegraph can upgrade their instances today, and Sourcegraph Cloud users will receive this update within the coming days.
```

Comment:
5.x.x is a placeholder?

Reply:
Yeah it was a placeholder since we don't get the release number until its cut now. But it looks like it got cut 15m ago! 5.4.5071